assets/swg-button.css

```diff
@@ -257,3 +257,17 @@
 .swg-button-dark:lang(zh-tw):after {
   background-image: url("i18n/b-zh-tw-dk.svg");
 }
+
+.swg-smart-button {
```

Comment:
Please add a comment here that all "button" styles will be defined inside the iframe.

Reply:
Done